documentation/examples/04-events/00-dom-events/App.svelte

```diff
@@ -7,7 +7,7 @@
 	}
 </script>
 
-<div on:mousemove={handleMousemove}>
+<div on:mousemove={handleMousemove} role='button' tabindex='0'>
```

Comment:
This should stay as is - the tutorial is showing reacting to mouse movements. Adding `role="button" tabindex="0"` doesn't help here. It would be better to ignore the warning instead, at least for now.